Makefile

```diff
@@ -2,6 +2,8 @@ NAME      := spdk
 SRC_EXT   := gz
 SOURCE     = https://github.com/spdk/$(NAME)/archive/v$(VERSION).tar.$(SRC_EXT)
 
+PR_REPOS := dpdk@PR-9
```

Comment:
This of course needs to be removed before landing.

Reply:
done